Utils/Dataflow/095_datasetInfoAMI/amiDatasets.py

```diff
@@ -39,13 +39,13 @@ def main(argv):
     stage.set_input_message_type(messageType.JSON)
```

Comment:
@mgolosova 
A small typo in the description: "So IN is not that..." - third paragraph. :)

Reply:
Fixed, thank you!